joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreator.scala

```diff
@@ -39,8 +44,25 @@ class AstCreator(val relativeFileName: String, val parserResult: ParserResult, v
   }
 
   private def astForCompilationUnit(cu: DotNetNodeInfo): Seq[Ast] = {
-    val imports    = cu.json(ParserKeys.Usings).arr.map(createDotNetNodeInfo).toSeq // TODO: Handle imports
+    val imports  = cu.json(ParserKeys.Usings).arr.map(createDotNetNodeInfo).toSeq // TODO: Handle imports
+    val name     = s"${parserResult.filename}"                                    // TODO: Find fullyQualifiedPackage
+    val fullName = s"$relativeFileName:$name"
+    val fakeGlobalMethodForFile =
```

Comment:
Do we really need one of these in C#? I don't think expressions can be outside of methods or field initializers. I may be wrong

Reply:
C# has global variable declarations, that is outside a namespace.

Reply:
Ok I see, then we will need one, but then I think let's continue developing it as a separate ticket